src/ios/CDVCamera.m

```diff
@@ -59,6 +59,59 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 @implementation CDVPictureOptions
 
+/*
+ API:
+ Camera:
+ - CameraPopoverHandle:
```

Comment:
Also, why do we have the API documented here ? If the documentation is lacking, shouldn't we add it to the README.md that shows up so that app developers can also take advantage of it ?